library/general/test/popup_test.rb

```diff
@@ -14,26 +14,30 @@
   end
 
   describe ".Feedback" do
-    before do
-      expect(ui).to receive(:OpenDialog)
-      expect(ui).to receive(:CloseDialog)
-      allow(ui).to receive(:BusyCursor)
-      allow(ui).to receive(:GetDisplayInfo).and_return({})
+    context "when arguments are good" do
+      before do
+        expect(ui).to receive(:OpenDialog)
```

Comment:
(Most of this hunk is whitespace. See https://github.com/yast/yast-yast2/pull/485/files?w=0 )